acceptance/localcluster/localcluster.go

```diff
@@ -564,9 +564,11 @@ func (l *Cluster) stop() {
 				log.Infof("node %d: stderr at %s", i, file)
 			}
 		} else if crashed {
+			// Don't remove crashed containers - could be useful for post-mortem.
```

Comment:
I just worry that if you default this behavior to true then we'll have various devs ending up with tons of old docker containers lying around. I don't think controlling this via an env variable is particularly onerous.